[ARGO-25] Stesura verbale del 2 maggio (#60)
Argo-swe/Docs: sources/documents/2_RTB/Verbali/Interni/VerbaleInterno_2024-05-02/capitolo-riunione.tex

\par Vengono infine discussi i prossimi lavori che riguardano la creazione di un secondo indice che faccia da indice intermedio tra la \glossario{query}\ e l'indice reale per estrarre l'informazione. Questo sarebbe utile perché arricchirebbe i risultati parziali con informazioni quali la descrizione della tabella e i vincoli comuni. Oltre a ciò c'è la volontà di testare l'inserimento del numero di colonne per confrontare e capire se il risultato finale sia ottimale.
\subsubsection{Progettista}
\par Il Progettista ha lavorato coordinandosi con il programmatore per la strutturazione di un dizionario dati in formato \glossario{JSON}\ che potesse essere più efficiente nell'interazione con \glossario{ChatGPT}. Sono stati citati diversi tipi di formato in linguaggio naturale che risultano essere più adatti gli \glossario{LLM} che verranno approfonditi in seguito. Infine è stato proposto di utilizzare il linguaggio \glossario{Python}\ per il \glossario{backend}\ vista la quantità già scritta di codice per la parte funzionale del progetto. 

\subsubsection{Ridistribuzione dei ruoli}

> \par Per il terzo sprint vengono distribuite nuovamente le ore ai vari membri del gruppo. Da questo sprint, le ore non si concentreranno solo su un ruolo, ma spazieranno su più ruoli per fornire sostegno a membri che necessitano di maggiori risorse e per evitare di assegnare solo un ruolo con poche ore rimanenti. In particolare:

Reviewer: ```suggestion
\par Per il terzo sprint  verranno distribuite nuovamente le ore ai vari membri del gruppo. Dal prossimo sprint, le ore non si concentreranno solo su un ruolo, ma spazieranno su più ruoli per fornire sostegno a membri che necessitano di maggiori risorse e per evitare di assegnare solo un ruolo con poche ore rimanenti. In particolare:
```
```suggestion
\par Per il terzo sprint vengono distribuite nuovamente le ore ai vari membri del gruppo. Da questo sprint, le ore non si concentreranno solo su un ruolo, ma spazieranno su più ruoli per fornire sostegno a membri che necessitano di maggiori risorse e per evitare di assegnare solo un ruolo con poche ore rimanenti. In particolare:
```

\begin{itemize}

> 	\item Vengono richieste più ore sul ruolo di progettista per dare una migliore definizione del lavoro al Programmatore. Vengono tuttavia assegnate più ore al Programmatore in vista di una ridistribuzione più mirata;

Reviewer: ```suggestion
	\item Sono state richieste più ore sul ruolo di progettista per dare una migliore definizione del lavoro al Programmatore. Allo stesso tempo saranno assegnate più ore al Programmatore e meno ad altri ruoli in vista di una ridistribuzione più mirata;
```
```suggestion
	\item Vengono richieste più ore sul ruolo di progettista per dare una migliore definizione del lavoro al Programmatore. Vengono tuttavia assegnate più ore al Programmatore in vista di una ridistribuzione più mirata;
```

> 	\item Vengono inoltre richieste più risorse al ruolo di Amministratore vista la mole di lavoro che altrimenti dovrebbe affrontare un solo membro. Vengono quindi assegnate altre ore a 2 membri del gruppo, in modo da offrire assistenza. 

Reviewer: ```suggestion
	\item Vengono inoltre richieste più risorse al ruolo di Amministratore vista la mole di lavoro che altrimenti dovrebbe affrontare un solo membro. Saranno quindi assegnate 2  ore aggiuntive ad altri membri del gruppo  in modo da offrire assistenza all'amministratore con più ore.
```

Author: Qui va lasciata la versione originale o mantenuto quel significato
